services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/credentials/AbstractCredentialsManagementService.java

```diff
@@ -167,7 +170,7 @@ public void setTenantInformationService(final TenantInformationService tenantInf
                                     resourceVersion,
                                     span));
                 })
-                .otherwise(t -> DeviceRegistryUtils.mapErrorToResult(t, span));
```

Comment:
`DeviceRegistryUtils.mapErrorToResult` can now be removed.

Reply:
I have pushed corresponding changes